worker/uniter/relation/resolver_test.go

```diff
@@ -514,7 +539,7 @@ func (s *relationResolverSuite) TestHookRelationChangedSuspended(c *gc.C) {
 	relationsResolver := relation.NewRelationResolver(r, nil)
 	op, err := relationsResolver.NextOp(localState, remoteState, &mockOperations{})
 	c.Assert(err, jc.ErrorIsNil)
-	assertNumCalls(c, &numCalls, numCallsBefore)
+	assertNumCalls(c, &numCalls, numCallsBefore+2) // Refresh/Life calls made by the resolver prior to emitting a RelationDeparted hook
```

Comment:
Would this be clearer called out as a separate line:
// relationsResolver.NextOp will notice it wants to fire relation-departed, and makes 2 calls
// to the controller to check if it is the unit that is departing.
expectedNumCalls := numCallsBefore + 2

Reply:
I am really keen on spending some time on the next cycle (if we have enough capacity) to rewrite all these tests to use mocks. Then we won't have to resort to this hacky approach where adding a new call anywhere in our logic means that we need to bump the expected call count in a bunch of unrelated tests...